src/desktop/components/artist_page_cta/view.coffee

```diff
@@ -35,12 +35,14 @@ module.exports = class ArtistPageCTAView extends Backbone.View
     @alreadyDismissed = false
     @signupIntent = 'Artist CTA Banner'
     @afterAuthPath = window.location
-    @$window.on 'scroll', _.throttle(@maybeShowOverlay, 200)
+    @$window.on 'scroll', _.once(@maybeShowOverlay)
     mediator.on 'clickFollowButton', @fullScreenOverlay
     mediator.on 'clickHeaderAuth', @fullScreenOverlay
```

Comment:
@mzikherman I was also wondering if the `clickFollowButton` and `clickHeaderAuth` handlers could be depreciated because it seems those are being handled by the new modal.

Reply:
Yes, looks like `clickHeaderAuth` isn't referenced anywhere. `clickFollowButton` is referenced by the old Follow button, but since the artist page is fully in Reaction (and uses the Reaction follow button -> modal), I think they both can be removed here!